src/nxt_controller.c

```diff
@@ -2432,6 +2430,9 @@ nxt_controller_conf_store(nxt_task_t *task, nxt_conf_value_t *conf)
     nxt_buf_t      *b;
     nxt_port_t     *main_port;
     nxt_runtime_t  *rt;
+    time_t         rawtime;
+    struct         tm *timeinfo;
+    u_char         buffer[25];
```

Comment:
Variables should be sorted in christmias tree order and variable types of the same length should be sorted alphabetically, yeah, I know, I didn't make the rule...